test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
   (<any> drawer)._svgElementName = "circle";
   return drawer;
 }
 
 describe("Drawers", () => {
   describe("Abstract Drawer", () => {
-    var oldTimeout: any;
-    var timings: number[] = [];
-    var svg: d3.Selection<void>;
-    var drawer: Plottable.Drawer;
+    let oldTimeout: any;
+    let timings: number[] = [];
+    let svg: d3.Selection<void>;
+    let drawer: Plottable.Drawer;
```

Comment:
Note that the `let` and the `const` does not return to the object side of the expressions but specifically the reference side of the expressions.  Returning a `const` and then using it as a `let` is I think an incorrect interpretation of what's going on here.

If I were to name `createMockDrawer()`'s `drawer` variable to be `mockDrawer`, would this still run into the same issues?